src/Blocks/custom/load-more/components/load-more-editor.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
why do you need load more block?

you are never going to use it as a block. please remove it